librz/type/type.c

```diff
@@ -982,12 +995,18 @@ static char *type_as_pretty_string(const RzTypeDB *typedb, const RzType *type, c
 	rz_strbuf_append(buf, typename_str);
 
 	if (btype) {
+		bool no_memb = false; // to check if no members are present
```

Comment:
It's probably better to do that using `rz_vector_len(btype->struct_data.members)` etc instead.